recipes/run_test.py

```diff
@@ -0,0 +1,2 @@
+import pyedflib
```

Comment:
Please delete this file and instead import `pyedflib` under the `tests/imports` tag.